pkg/scheduler/plugins/elasticquota/core/quota_operator.go

```diff
@@ -0,0 +1,83 @@
+/*
+Copyright 2022 The Koordinator Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package core
+
+import (
+	"encoding/json"
+
+	"sigs.k8s.io/scheduler-plugins/pkg/apis/scheduling/v1alpha1"
```

Comment:
The order of import packages still needs to be paid attention to. Referring to the style of most of the koordinator's code, adjust it in the order of std, third-party and koordinator.